shared/mpapiResponse.ts

```diff
@@ -20,3 +21,22 @@ export const AppSubscriptionSoftOptInIds: string[] = [
 	SoftOptInIDs.SimilarProducts,
 	SoftOptInIDs.SupporterNewsletter,
 ];
+
+export enum AppStore {
+	IOS,
+	ANDROID,
+	UNKNOWN,
+}
+
+export function determineAppStore(subscription: AppSubscription) {
+	if (subscription.productId.includes('guardian.subscription')) {
+		return AppStore.ANDROID;
+	} else if (
+		subscription.productId.includes('guardian.gia') ||
+		subscription.productId.includes('guardian.gla')
+	) {
+		return AppStore.IOS;
+	} else {
+		return AppStore.UNKNOWN;
+	}
+}
```

Comment:
It's a stylistic preference, but each condition has a `return` so you don't really need `else`. (I think it might look a little cleaner though.)